pom.xml

```diff
@@ -9,16 +9,97 @@
         <artifactId>jenkins</artifactId>
         <version>1.51</version>
     </parent>
-    <packaging>pom</packaging>
-    <groupId>io.jenkins.plugin-management</groupId>
-    <artifactId>jenkins-plugin-management-parent-pom</artifactId>
-    <version>1.0-SNAPSHOT</version>
+
     <properties>
+        <java.level>8</java.level>
+        <!-- Version of Maven Components -->
+        <maven.version>3.5.0</maven.version>
+        <powermock.version>2.0.2</powermock.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
+
+    <groupId>io.jenkins.plugin-management</groupId>
+    <artifactId>jenkins-plugin-management-parent-pom</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    <url>https://github.com/jenkinsci/plugin-installation-manager-tool</url>
+
     <modules>
-        <module>plugin-management-cli</module>
         <module>plugin-management-library</module>
+        <module>plugin-management-cli</module>
     </modules>
 
+    <licenses>
+        <license>
+            <name>The MIT license</name>
+            <url>http://www.opensource.org/licenses/mit-license.php</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+    <dependencies>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-module-junit4</artifactId>
+            <version>${powermock.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-api-mockito2</artifactId>
+            <version>${powermock.version}</version>
+            <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>
+                        org.objenesis
+                    </groupId>
+                    <artifactId>objenesis</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+    </dependencies>
+
+
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-checkstyle-plugin</artifactId>
+                    <version>3.0.0</version>
+                    <dependencies>
+                        <dependency>
+                            <groupId>com.puppycrawl.tools</groupId>
+                            <artifactId>checkstyle</artifactId>
```

Comment:
https://nvd.nist.gov/vuln/detail/CVE-2019-9658

Reply:
Also added updating the plugin to [JENKINS-58078].